package.json

```diff
@@ -32,6 +33,7 @@
     "build": "run-p build:*",
     "build:es": "cross-env BABEL_ENV=es babel src -d es",
     "build:commonjs": "cross-env BABEL_ENV=commonjs babel src -d lib",
+    "build:umd": "cross-env BABEL_ENV=umd rollup -c",
```

Comment:
it would be great to provide 2 umd builds, second one would be minified (based on `NODE_ENV=production/development` - this will require adding uglifyjs plugin and using it conditionally in the `rollup.config.js`)